client/my-sites/site-settings/theme-setup-dialog/index.jsx

```diff
@@ -127,4 +129,6 @@ const mapStateToProps = ( state ) => {
 	};
 };
 
-export default connect( mapStateToProps, { toggleDialog, runThemeSetup } )( ThemeSetupDialog );
+export default connect( mapStateToProps, { toggleDialog, runThemeSetup: runThemeSetupAction } )(
+	localize( ThemeSetupDialog )
```

Comment:
This `localize` is new. I guess it is intentional, right?

Reply:
It's not new; it was being applied in line 115, by rewriting the class. The linter didn't like that, so I moved it into the export instead.